### ISSUE TITLE
using makefiles

### ISSUE THREAD

**Issue by eflasch** [biostars/biostar-handbook-issues#95]
In the Using Makefiles section of the handbook:
If I write "foo:" and press enter I get a "command not found"
How do I get to the next line  "tab echo Hello John?" without triggering a command not found
Thank you

**Comment by ialbert** [comment]
the command you need to enter is

   make foo

not just `foo`

**Comment by eflasch** [comment]
I do appreciate your answer, but when I type "make foo:" , I get 
make: *** No rule to make target 'foo:'.  Stop. 

Thank you

**Comment by ialbert** [comment]
This error means that your file does not contain the rule. 

There is a new feature to the book that we are testing out. We called this "command casts", where we demonstrate the usage and potential errors that you might experience as an animated bash shell capture. 

I reworked this chapter a bit to add these command casts in response to this ticket. I demonstrate what causes the error and how to fix them:

Take a look at the page (make sure to reload it) to see the animations:

https://www.biostarhandbook.com/books/scripting/makefiles.html

I hope these will help you see what the expected behaviors are.
